docs/reference/synonyms/apis/put-synonyms-set.asciidoc

```diff
@@ -0,0 +1,215 @@
+[[put-synonyms-set]]
+=== Put synonyms set
```

Comment:
```suggestion
=== Create or update synonyms set
```

Reply:
The convention has been to say "create or update" instead of "put" (see https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-put-template.html for example)